programs/mango-v4/src/accounts_ix/openbook_v2_create_open_orders.rs

```diff
@@ -0,0 +1,54 @@
+use anchor_lang::prelude::*;
+
+use crate::error::*;
+use crate::state::*;
+use openbook_v2::{
+    program::OpenbookV2,
+    state::{Market, OpenOrdersAccount},
+};
+
+#[derive(Accounts)]
+#[instruction(account_num: u32, open_orders_count: u8)]
+pub struct OpenbookV2CreateOpenOrders<'info> {
+    #[account(
+        constraint = group.load()?.is_ix_enabled(IxGate::OpenbookV2CreateOpenOrders) @ MangoError::IxIsDisabled,
+    )]
+    pub group: AccountLoader<'info, Group>,
+
+    #[account(
+        mut,
+        has_one = group,
+        constraint = account.load()?.is_operational() @ MangoError::AccountIsFrozen
+        // owner is checked at #1
+    )]
+    pub account: AccountLoader<'info, MangoAccountFixed>,
+    pub owner: Signer<'info>,
+
+    #[account(
+        has_one = group,
+        has_one = openbook_v2_program,
+        has_one = openbook_v2_market_external,
+    )]
+    pub openbook_v2_market: AccountLoader<'info, OpenbookV2Market>,
+
+    pub openbook_v2_program: Program<'info, OpenbookV2>,
+    pub openbook_v2_market_external: AccountLoader<'info, Market>,
+
+    // initialized by this instruction via cpi to openbook_v2
+    #[account(
+        init,
```

Comment:
is it really up to us to init this account? I suspect it'd be the openbookv2 program that inits it!

Reply:
Yes i have moved it to unchecked account. we cannot init openbook_v2 programs account with anchor like this.